tmk_core/common/keyboard.h

```diff
@@ -57,7 +57,7 @@ static inline bool IS_RELEASED(keyevent_t event) { return (!IS_NOEVENT(event) &&
     .time = (timer_read() | 1)                          \
 }
 
-
+void make_ghost_check_array(void);
```

Comment:
This function does not need to be exposed to the header file, as it's not called from anywhere else.

Also see the comment above about making the function static.